### PR TITLE
add arm64

### PR DESCRIPTION
**Pull request by varnson**
fix https://github.com/hashicorp/consul-esm/issues/88
add arm64

### CODE REVIEW

**Comment by eikenb** [comment]
Hey @varnson, thanks for submitting this. 

To add arm64 it needs a few additional changes to exclude unsupported platforms. Probably easiest for you to just look at what consul-template does as it already has arm64 support and excludes all the necessary platforms (probably more than needed here).

https://github.com/hashicorp/consul-template/blob/91c3dd4189ad3d8d21130694f436e9706e3ccf5a/Makefile#L31-L32

You should be able to just specify the "arm64" exclusions just like it does in consul-template's Makefile, as it does the same comparison.

**Comment by varnson** [comment]
@eikenb
Thank you for your advice. I have added it.

**Comment by eikenb** [comment]
Sorry @varnson, but would you mind including the comment that was above the XC_EXCLUDE definition. Makefiles can be pretty dense and it is not obvious how that comparison works. Thanks.

File: Makefile [hunk 1]
```diff
@@ -24,8 +24,9 @@ GOMAXPROCS ?= 4
 
 # Default os-arch combination to build
 XC_OS ?= darwin freebsd linux netbsd openbsd solaris windows
-XC_ARCH ?= 386 amd64 arm
-XC_EXCLUDE ?= darwin/arm solaris/386 solaris/arm windows/arm netbsd/arm
+XC_ARCH ?= 386 amd64 arm arm64
+# XC_EXCLUDE "arm64" entries excludes both arm and arm64
+XC_EXCLUDE ?= darwin/arm solaris/386 solaris/arm windows/arm netbsd/arm darwin/arm64 freebsd/arm64 netbsd/arm64 openbsd/arm64 solaris/arm64 windows/arm64
```
Comment (eikenb) [inline Makefile:29]: Maybe that comment isn't very clear about this, but what it is trying to explain is that the XC_EXCLUDE list excludes arm entries without having them... that the arm64 entries are enough to exclude both arm and arm64 due to how it matches (it does sub-string matching and "darwin/arm64" matches both "darwin/arm64" and "darwin/arm" as "arm" is a sub-string of "arm64").

If you compare the XC_EXCLUDE list here to the one in consul-template you'll see it has no "arm" entries, just "arm64" entries and the same can be done here.

**Comment by eikenb** [comment]
Looks good. Thanks for the changes.

**Pull request merged**